packages/components/src/components/TaskRunStatus/TaskRunStatus.js

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { injectIntl } from 'react-intl';
+import React from 'react';
+import { ChevronRight24 as ChevronRight } from '@carbon/icons-react';
+import { ViewYAML } from '..';
+
+import './TaskRunStatus.scss';
+
+const TaskRunStatus = props => {
+  const { intl, reason, taskRun } = props;
+  const { status } = props;
+
+  return (
+    <div className="step-details">
+      <header
+        className="step-details-header"
+        data-status={status}
+        data-reason={reason}
+      >
+        <h2>
+          <ChevronRight className="status-icon" />
+          {taskRun.pipelineTaskName || taskRun.taskRunName}
+          <span className="status-label">
+            {intl.formatMessage({
+              id: 'dashboard.taskRun.status.notRun',
+              defaultMessage: 'Not run'
+            })}
+          </span>
+        </h2>
+      </header>
+      <div className="step-status">
+        <ViewYAML resource={taskRun.status} />
+      </div>
```

Comment:
wrap this in a tab so it's consistent with the step details and looks a bit more complete

```js
<Tabs>
  <Tab
    className="details-tab"
    label={intl.formatMessage({
      id: 'dashboard.taskRun.status',
      defaultMessage: 'Status'
    })}
  >
    <div className="step-status">...</div>
  </Tab>
</Tabs>
```

we'll expand this in future to include more info